### ISSUE TITLE
MOD LIST (not Load Order): Export / Copy to Text

### ISSUE THREAD

**Issue by Vyxenne** [Nexus-Mods/Vortex#5694]
Firstly, I'm a fan- Vortex has worked very well for me (I was leery after the whole NMM 0.60 debacle but Vortex imported my NMM mods, and with only a few tweaks, ran it better than NMM ever did.) So thank you for creating it.

I have seen this feature requested over and over, but no actual resolution that I can find except that the OP and/or user comments invariably turn to "load order" instead of "mod list," making it unclear just which of the _**two not-identical proposals**_ you are responding to. Also, I probably didn't find them all, so there may have been a definitive resolution/response from the Dev Team and I just failed to find it. Load Order is not the same thing as Mod List. This and the other similar requests are about Mod Lists, NOT Load Orders.

Please, we know we can easily get our load orders from plugins.txt and loadorder.txt, LOOT's built-in Copy Load Order to Clipboard feature and possibly other more obscure sources. That is not the same as this request. Won't you please consider enabling Vortex to simply Export or Copy to Clipboard our _**mod lists**_, subject to the current filters (All, Enabled, Disabled, etc.).

If you are unable or unwilling to consider that, would you be so kind as to tell us where else we can get the information so that we can share our mod lists with each other without taking 762 screenshots of Vortex MODS pane displays? :-P

Thank you.

**Describe the solution you'd like**
A button to press that copies the current mod list subject to the filters currently applied

**Describe alternatives you've considered**
With almost 300 mods, no viable alternatives present themselves AFAIK.

**Comment by TanninOne** [comment]
There is an extension for this: https://github.com/Garethp/Vortex-Modlist-Backup

**Comment by Vyxenne** [comment]
Thank you. I installed it, but there is no obvious way to run it, nor is there any yxy or json file created on Vortex launch, nor is there any documentation on the github page where the extension is housed.

TL; DR: It's installed and doing nothing, nor can it be manually run.

**Comment by notho** [comment]
Garethp's 'backup does not work, (Sorry Garethp).  My profile has 33 mods, yet it prints out over 400 mods from all profiles.  I understand you are avoiding allowing an export because you don't want people passing around mod lists - ok, I'll move on to another manager.  I really liked vortex but this is becoming a deal killer.
